src/status_im2/common/keychain/events.cljs

```diff
@@ -79,33 +78,88 @@
       (if can-save?
         (keychain/get-credentials
          (str key-uid "-auth")
-         #(callback (if % (oops/oget % "password") auth-method-none)))
+         (fn [value]
+           (callback (if value (oops/oget value "password") auth-method-none))))
```

Comment:
should "password" be in a private def ? 🤔

Reply:
Not the same, but still related, I prefer to pass keywords to `oget` because they only need to be allocated once. It's not important really, just a nano optimization, but feels more idiomatic to me 🤷🏼